src/components/tabbed-list/family-member-page.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect, useCallback } from "react";
+import PropTypes from "prop-types";
+import "./../../styles.scss";
+import "./tabbed-list.scss";
+import classNames from "classnames";
+import { TabbedListItem } from "./tabbed-list-item";
+
+export const FamilyMemberPage = () => {
+  const [members, setMembers] = useState([]);
+  const [listItems, setListItems] = useState([]);
+  const [dropdownActive, setDropdownActive] = useState(false);
+  const [selected, setSelected] = useState(null);
+
+  useEffect(() => {
+    //TODO make api call --- mocked result
+    const result = [
+      { id: 1, name: "Camelia Nistor" },
+      { id: 2, name: "Andrei Nistor" }
+    ];
+    setMembers(result);
+    setSelected(result[0]);
+  }, []);
+
+  useEffect(() => {
+    //TODO make api call --- mocked result
+    const result = [
+      {
+        id: 1,
+        color: "#0000ff",
+        date: new Date(),
+        resultText: "Rezultat Formular - esti in siguranta daca ramai in casa",
+        symptomText: "Simptome - nu prezinti simptome specifice COVID-19",
+        onClick: () => {
+          alert("hello");
+        }
+      },
+      {
+        id: 2,
+        color: "#ff0000",
+        date: new Date(),
+        resultText: "Rezultat Formular - esti in siguranta daca ramai in casa",
+        symptomText: "Simptome - prezinti simptome specifice COVID-19",
+        onClick: () => {
+          alert("hello");
+        }
+      }
+    ];
+
+    setListItems(result);
+  }, [selected]);
+
+  const triggerDropdown = useCallback(() => {
+    setDropdownActive(true);
+  }, []);
+
+  const handleSelect = useCallback(m => {
+    setSelected(m);
+    setDropdownActive(false);
+  }, []);
+
+  return (
+    <div className="tab-page">
+      <div className="tli-dropdown-container">
+        <div className="vis-text">Vizualizeaza pentru:</div>
+        <div className="action-container">
+          <div
+            className={classNames("dropdown", { "is-active": dropdownActive })}
```

Comment:
Shouldn't we use a select element for the dropdown? I think it would be easier to have all the functionality built in by default, like closing itself when clicking outside

Reply:
Using a `select` tag is better for accessibility too.